assign_nodes.py

```diff
@@ -22,6 +22,14 @@
 pd.set_option('display.max_rows', None)
 
 
+#Added here to avoid circular import
+def get_label(labels,value,kg_type):
```

Comment:
I got an error here: 

<img width="1219" alt="image" src="https://github.com/UCDenver-ccp/Cartoomics/assets/36578655/d9cb44ea-8e90-4a83-a8d9-62e3343d4f5c">

Reply:
I believe this is fixed. I didn't realize some ncbi gene ID's wouldn't be in PKL, which is hard to know what to index them to.

Reply:
This works now. It seems the problem was with some pseudogenes. We should consider how to deal with this. My thinking is to allow the user to skip this gene if it is not represented. I'll make an issue.